repairnator/docker-images/k8s-pipeline-dockerimage/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM maven:3.6.1-jdk-8
+
+LABEL Description="Repairnator Pipeline docker image for k8s" Vendor="Spirals" Version="0.0.0"
+LABEL maintainer="Long Zhang <longz@kth.se>"
+
+VOLUME ["/var/log/"]
+
+
+COPY projects_to_ignore.txt /root/
+COPY logback.xml /root/
+COPY configure_git.sh /root/
+COPY build_repairnator.sh /root/
+COPY install_python_stomp.sh /root/
+COPY pipeline_launcher.sh /root/
+COPY pipeline_worker.py /root/
+COPY version.ini /root/
+
+RUN touch /root/z3_for_linux
+RUN chmod +x /root/*.sh
+RUN echo "Europe/Paris" > /etc/timezone
+RUN apt-get update
+RUN apt-get install cloc -y
+RUN /root/configure_git.sh
+RUN /root/install_python_stomp.sh
+RUN /root/build_repairnator.sh
+
+# Can add more tools or remove some. Need at least 1 tool otherwise can be left as it is.
+ENV REPAIR_TOOLS=NopolAllTests,AstorJMut,NPEFix
```

Comment:
When I test this Docker image in our own k8s cluster, I get the following error:
```
WARNING:root:stderr
WARNING:root:Error: Unexpected argument: NopolAllTests,AstorJMut,NPEFix
```
It seems that the latest version of repairnator pipeline does not take the argument in this format. Or you need to specify them with a flag?

Reply:
I think we can re-set that in the yaml file since I specified this in the Dockerfile as an env variable. :)